lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -183,4 +186,8 @@ export class ManagedIdentityApplication {
             );
         }
     }
+
+    public getManagedIdentitySource(): AzureIdentitySdkManagedIdentitySourceNames {
```

Comment:
It's a public API so it needs comments. The comments should clearly state that this API is for Azure SDK.

Also, maybe it could live somewhere else? Like some helper that we don't document?

Reply:
I'll add comments.

However, it needs to live here because only `ManagedIdentityApplication.ts` has a reference to `ManagedIdentityClient.ts`, which is not part of the public api.